templates/web/base/auth/2faform.html

```diff
@@ -0,0 +1,25 @@
+[% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title = loc('Confirm account') %]
+
+  <div class="confirmation-header confirmation-header--phone">
+    [% IF incorrect_code %]
+        <h1>[% loc('Sorry, that wasn&rsquo;t the correct code') %]</h1>
+        <p>[% loc('Try again') %]:</p>
+    [% ELSE %]
+        <h1>[% loc("Nearly done! Now check your phone&hellip;") %]</h1>
+        <p>[% loc("Please generate a 2FA code and enter it below:") %]</p>
```

Comment:
Not sure that's the correct title to use :)

Reply:
I assumed most people will have the Authenticator app on their phone, so that is enough right for me :)